### PR TITLE
Removed Distinct

### PR DESCRIPTION
**Pull request by mickel1138**
This fixed the JSON type break, I'm not sure why the `Distinct` was added. Please try it out.

### CODE REVIEW

**Review by fightingtexasaggie — APPROVED** [review]
I can't see any duplication, but the original comment was made about preventing dupes and I'm a little concerned as to why it was put in. If there was a way to detect and disable the distinct just on json types, there would be less possible exposure of the change, however, I don't see anything breaking as I'm testing. There are some 29 references to entityHandler's getBy method across the codebase though.

File: src/Handler/EntityHandler.php [hunk 1]
```diff
@@ -173,8 +173,7 @@ public function getBy(
         }
         // Filter by Funding Organization is needed.
         $this->filterByFundingOrganization($qb, $entityClass);
-        // Eliminate duplicates.
-        $qb->distinct();
+
```
[the remark this answers is not in the document]
Comment (mickel1138) [reply src/Handler/EntityHandler.php:176]: Yes you can. But the problem is (see PELAGOS ticket) that JSON does not have an Equalizer, and DISTINCT uses this to compare uniqueness.

**Review by praneethpr — DISMISSED** [review]

**Review by praneethpr — DISMISSED** [review]

**Review by fightingtexasaggie — DISMISSED** [review]
tested. I agree this is a lot safer.

File: src/Handler/EntityHandler.php [hunk 2]
```diff
@@ -151,7 +151,8 @@ public function getBy(
         array $criteria,
         array $orderBy = array(),
         array $properties = array(),
-        $hydrator = null
+        $hydrator = null,
+        $distinct = true
```
Comment (mickel1138) [inline src/Handler/EntityHandler.php:155]: Add type `bool`

**Review by fightingtexasaggie — APPROVED** [review]

**Pull request merged**